src/types.rs

```diff
@@ -614,7 +627,7 @@ new_secret_type![
 
 #[cfg(test)]
 mod tests {
-    use crate::{ClientSecret, CsrfToken, PkceCodeChallenge, PkceCodeVerifier};
+    use super::*;
```

Comment:
I'd prefer to avoid wildcard imports except from `prelude` or similar modules since they can make dependencies harder to see and grep for